application/src/main/resources/db/V5__Add_Warn_System.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE warn_system
```

Comment:
Your 3 sql files should be in a single migration file and not 3 different migration versions...